main.js

```diff
@@ -1,19 +1,15 @@
-const hamMenu = document.getElementById("hamMenuImg");
-const closeMenu = document.getElementById("closeMenuImg");
-const respMenu = document.getElementById("responsiveMenu");
-
+const hamMenu = document.getElementById('hamMenuImg');
+const closeMenu = document.getElementById('closeMenuImg');
+const respMenu = document.getElementById('responsiveMenu');
 
 function toggle() {
-    respMenu.style.display = "flex";
+  respMenu.style.display = 'flex';
 }
 
 function close1() {
-    respMenu.style.display = "none";
+  respMenu.style.display = 'none';
 }
 
 hamMenu.addEventListener('click', toggle);
 closeMenu.addEventListener('click', close1);
 respMenu.addEventListener('click', close1);
```

Comment:
- [OPTIONAL] I think it's better to close the mobile menu just when the user clicks on some menu option or the close button. Your code is closing the menu when the user clicks on any part of the menu. This change will create a better UX and UI.